Makefile

```diff
@@ -309,25 +328,42 @@ test-unit-cpp: build-development-image ## Run C++ unit tests
 		&& make -j 4 \
 		&& make test"
 
-test-unit-python: build-release-image-python ## Run Python unit tests
+test-unit-python:
+
+	@ make build-development-image
+
+	@ make test-unit-python-standalone
+
+test-unit-python-standalone: ## Test with no build for CI
 
 	@ echo "Running Python unit tests..."
 
 	docker run \
```

Comment:
This now uses the dev image to run the python tests. It was using the release image, but because image caching wasn't happening, it was build the docker image as well as the Cpp/Python code. Now it just builds the code.